.github/workflows/ci.yml

```diff
@@ -320,8 +320,6 @@ jobs:
         uses: actions/checkout@v2
         with:
           persist-credentials: false
-          # Needs to fetch all history to be able to verify changelog generation
-          fetch-depth: 0
```

Comment:
Do we not need this anymore?

Reply:
We don't :)

Reply:
Fetching is done by the script - we are fetching from specially prepared read only remote https-something-for-providers - this way we make sure we fetch latest main always - no matter where we branched off - this way we will be warned in case there are already some changes merged that we did not rebase to